core/config/v2/types.go

```diff
@@ -50,6 +50,10 @@ type Core struct {
 	Sentry *Sentry
 }
 
+func (core Core) ValidateConfig() (err error) {
+	return
+}
```

Comment:
Is it necessary to define this when it's empty/no-op? I thought I rigged it to work without these :thinking:

Reply:
It was initially a little confusing to me where the actual validation is supposed to happen so I added it to make it clear for future readers. I can remove from this PR if it's not a good practice to keep empty functions around.

Reply:
Or just add a todo linked back to this story to be filled in?